index.js

```diff
@@ -507,6 +515,60 @@ app.get('/api/user/:id', async (req, res) => {
   }
 });
 
+
+
+// register new user
+app.post('/api/register', async (req, res) => {
+  var { name, email, phone_number, password } = req.body;
+
+  if (email == "" || name == "" || password == "") {
```

Comment:
```suggestion
  if (!email || !name || !password) {
```
I think we should check with `!` in case the fields aren't included at all (one of these expressions will still return true if it's an empty string)